### PR TITLE
Refactors roundstart checks slightly

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/controllers/master.dm [hunk 1]
```diff
@@ -216,7 +216,7 @@ GLOBAL_VAR_INIT(CURRENT_TICKLIMIT, TICK_LIMIT_RUNNING)
 	//	local vars rock
 
 	// Schedule the first run of the Subsystems.
-	round_started = world.has_round_started()
+	local_round_started = SSticker.HasRoundStarted()
```
Comment (MrStonedOne) [inline code/controllers/master.dm]: keep the world.has_round_started() proc and have it forward to ssticker.

the point of it is to keep the mc modular enough somebody could rip it out and implement it in their project regardless of if the project is ss13

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -651,11 +651,11 @@ SUBSYSTEM_DEF(ticker)
 		return
 	INVOKE_ASYNC(SSmapping, /datum/controller/subsystem/mapping/.proc/maprotate)
 
+/datum/controller/subsystem/ticker/proc/HasRoundStarted()
```
Comment (optimumtact) [inline code/controllers/subsystem/ticker.dm]: these should be macros

File: code/world.dm [hunk 3]
```diff
@@ -314,3 +314,6 @@
 		s += ": [jointext(features, ", ")]"
 
 	status = s
+
+/world/proc/has_round_started()
```
Comment (optimumtact) [inline code/world.dm]: macro

Comment (Cyberboss) [inline code/world.dm]: MSO specifically said to keep this proc

File: code/controllers/subsystem/ticker.dm (hunk 2 above)
Comment (Cyberboss) [inline code/controllers/subsystem/ticker.dm]: y tho? It's not a performance critical check

Comment (optimumtact) [inline code/controllers/subsystem/ticker.dm]: reeeeeeee

**Pull request merged**